pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -188,10 +188,12 @@
     "version": "2.1.2"
   },
   "cloudstack": {
-    "owner": "terraform-providers",
+    "owner": "cloudstack",
+    "provider-source-address_broken": "registry.terraform.io/cloudstack/cloudstack",
```

Comment:
The version on the registry doesn't have a corresponding tag in the source repo so doing this just so it's skipped by the update script.

Reply:
Let's remove all the broken packages.